<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Настройки обработки видео по умолчанию:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Чувствительность</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="sensitivity_settings"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Отступ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="indent_settings"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Шаг</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="step_settings"/>
     </item>
     <item>
      <widget class="QPushButton" name="info_btn">
       <property name="text">
        <string>Инфо</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Путь сохранения файлов по умолчанию:</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLineEdit" name="default_path"/>
     </item>
     <item>
      <widget class="QPushButton" name="choose_path_btn">
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="can_edit">
     <property name="text">
      <string>Разрешить ручное редактирования пути</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="log">
     <property name="text">
      <string>Записывать лог файл</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="save_btn">
     <property name="text">
      <string>Сохранить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
